gallery/src/components/NxModal/NxModalPage.tsx

```diff
@@ -182,6 +182,44 @@ export default function NxModalPage() {
           Note: Placing content into the modal which exceeds its horizontal bounds is not supported. The resulting
           layout is unspecified.
         </NxInfoAlert>
+        <h3>Accessibility</h3>
+        <p className="nx-p">
+          Elements with the <code className="nx-code">dialog</code> role are supposed to have a label specified by the
```

Comment:
It probably isn't obvious to the reader that `NxModal` has the `dialog` role by default.

Reply:
> It probably isn't obvious to the reader that `NxModal` has the `dialog` role by default.

Ah, now it should be: https://github.com/sonatype/sonatype-react-shared-components/pull/440/commits/f25b9117f3094c3bbaf8f2e5c6269dfe176fbb6e